backend/services/core/src/shared/database.ts

```diff
@@ -151,13 +177,28 @@ export class KnexDataSource extends DataSource<UserContext> {
       const members = await this.knex<Keycloak>('keycloak').select('member_id');
```

Comment:
Is there any reason in particular that the keycloak table is queried here and not the members table?

Reply:
My mistake, will fix

Reply:
Just re-writing into a single SQL with a join query instead of two